src/Pages/Jobs/PostNewJob.jsx

```diff
@@ -476,27 +522,21 @@ const PostNewJob = () => {
               />
             </Grid>
             <Grid item xs={12} sm={6}>
-              <TextField
-                name="contactEmail"
-                label="Contact Email"
-                value={formData.contactEmail}
-                onChange={onChange}
-                fullWidth
-                size="small"
-                type="email"
-              />
-            </Grid>
-            <Grid item xs={12}>
               <TextField
                 name="externalLink"
                 label="External Link"
                 value={formData.externalLink}
                 onChange={onChange}
                 fullWidth
                 size="small"
-                error={formData.externalLink && !isValidURL(formData.externalLink)}
+                error={
+                  formData.externalLink.trim() !== "" &&
+                  !isValidURL(formData.externalLink)
+                }
                 helperText={
-                  formData.externalLink && !isValidURL(formData.externalLink)
+                  formData.externalLink.trim() === ""
+                    ? ""
+                    : !isValidURL(formData.externalLink)
```

Comment:
formData.externalLink.trim() === "" try to use instead of  Boolean(formData.externalLink) it will will convert in true or false

Reply:
good suggestion, fixed